packages/components/managed-data-table/src/data-table-settings/density-manager/density-manager.spec.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { screen, render } from '../../../../../../test/test-utils';
+import DensityManager from '.';
+import PrimaryButton from '../../../../buttons/primary-button';
+import SecondaryButton from '../../../../buttons/secondary-button';
+import {
+  WRAPPED_TEXT_VISIBLE,
+  SHOW_HIDE_ON_DEMAND,
+  DENSITY_DEFAULT,
+  DENSITY_COMPACT,
+} from './constants';
+
+const createTestProps = (customProps) => ({
+  isCondensed: false,
+  isWrappingText: false,
+  onDensityDisplayChange: jest.fn(),
+  onTextWrappingChange: jest.fn(),
+  onClose: jest.fn(),
+  ...customProps,
+});
+
+describe('DensityManager', () => {
+  it('not wrapping text', () => {
+    const props = createTestProps();
+    const { container } = render(<DensityManager {...props} />);
+    expect(
+      container.querySelector(`[value='${WRAPPED_TEXT_VISIBLE}']`)
```

Comment:
If the RadioButton would have a `AccessibleHidden` and a `Label` I would assume we could `getByLabelText('name').toHaveValue(WRAPPED_TEXT_VISIBLE)`. Not 100% sure but a thought.